pkg/version/version.go

```diff
@@ -38,7 +38,7 @@ var (
 	// It is customary to bump the version by incrementing the numeral after
 	// the `dev` tag. The SQL migration script name must correspond to the /new/ version.
 
-	Promscale                           = "0.7.0-beta.1.dev.3"
+	Promscale                           = "0.7.0-beta.1.dev.4"
```

Comment:
Any specific reason why are we changing the dev version on every PR? Doesn't this add friction as other PR's will collide with same `dev.X` tags and requires to rebase as other PR's get in with the same tag?

Reply:
As I understand it, that last number in the `dev.X` controls which SQL scripts get run from the upgrade folder. In other words, if you add a new upgrade script (like 4-drop_get_operation.sql) then you have to increase the version to `dev.4` or your upgrade script won't run.

Reply:
I could be mistaken, and/or not answering your question exactly.